contrib/asff.tpl

```diff
@@ -82,7 +82,7 @@
         {
             "SchemaVersion": "2018-10-08",
             "Id": "{{ $target }}/{{ .ID }}",
-            "ProductArn": "arn:aws:securityhub:{{ env "AWS_REGION" }}::product/aquasecurity/aquasecurity",
+            "ProductArn": "arn:aws:securityhub:{{ env "AWS_REGION" }}:{{ env "AWS_ACCOUNT_ID" }}:product/{{ env "AWS_ACCOUNT_ID" }}/aquasecurity",
```

Comment:
@AndrewCharlesHay could you help me to clarify why we should replace `aquasecurity` in this place?

i took a loot at your link from the issue, and found next definition of resource type `product`:
```
arn:${Partition}:securityhub:${Region}:${Account}:product/${Company}/${ProductId}
```
so my first thought is that `ProductArn` should be next:
```
"ProductArn": "arn:aws:securityhub:{{ env "AWS_REGION" }}:{{ env "AWS_ACCOUNT_ID" }}:product/aquasecurity/trivy",
```

isn't it?

Reply:
Yeah that does make more sense. I was referencing [the default arn](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-custom-providers.html#securityhub-custom-providers-bfi-reqs) but yours is better